packages/cvxpy-base/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: cvxpy-base
```

Comment:
Question: What is the difference between cvxpy and cvxpy-base?

Reply:
Mostly the dependencies. CVXPY requires the set of solvers necessary to be able to solve problems out of the box, e.g. ECOS, OSQP, SCS and Clarabel. cvxpy-base doesn't require any of the solvers, but still has the interfaces needed to communicate with any solver that's installed.

In other words, if you see CVXPY as a compiler from user written models to solver models, the CVXPY package depends on the solvers so you can compile and solve the model. CVXPY-base only has the compiler.